fuzzers/031-cmt-mmcm/write_mmcm_reg.py

```diff
@@ -0,0 +1,300 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2017-2020  The Project X-Ray Authors.
+#
+# Use of this source code is governed by a ISC-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/ISC
+#
+# SPDX-License-Identifier: ISC
+import argparse
+
+REGISTER_LAYOUT = {
+    'CLKOUT1': [
+        ('LOW_TIME', 6),
+        ('HIGH_TIME', 6),
+        ('OUTPUT_ENABLE', 1),
+        ('PHASE_MUX', 3),
+    ],
+    'CLKOUT2': [
```

Comment:
This layout differs across clock outputs. CLKOUT[0], CLKOUT[5] and CLKFBOUT have different bit fields related to fractional divider.

Reply:
Good catch, fixed.